docs/modelserving/logger/logger.md

```diff
@@ -180,6 +180,34 @@ predictor:
     storageUri: gs://kfserving-examples/models/sklearn/1.0/model
 ```
 
+## Inference Logger with Inference Service Annotation
+
+The inference service annotations can be included in the log message by specifying the annotation keys in the `metadataAnnotations` field 
+of the InferenceService CRD. These annotations are included in the CloudEvent extension attribute `annotations`.
```

Comment:
@tylerhyang Can you provide an example for cloud event showing the extension attribute ?